HTTP_200/templates/about.html

```diff
@@ -164,7 +164,7 @@ <h4><a href="http://hackncs.com/" target="_blank">Nibble Website</a></h4>
       <div class="row">
         <div class="footer-links">
           <div class="col col-3">
-            <h4>JSS Library</h4>
+            <h4><a href="https://ndl.iitkgp.ac.in/" target="_blank">National Digital Library of India</a></h4>
```

Comment:
use this link here. `http://jssaten.ac.in/library/`

Reply:
Okay sir.